metagov/metagov/core/views.py

```diff
@@ -1,8 +1,15 @@
+"""
+This module contains public-facing views that Metagov exposes on behalf of all drivers, both external
+drivers and Django app drivers.
```

Comment:
Hmm ok, with the way I've been restructuring PolicyKit, I wouldn't think that the 'Django app drivers' need any views or URLs. They just define their own views and use the MetagovAppHandler to process requests (for "public endpoints" i.e. oauth and webhooks). That's how I've implemented it so far anyway.

Reply:
Whoops, yeah, that's totally what we agreed on. Move that stuff over too.